.github/actions/run-backend-tests/action.yml

```diff
@@ -102,6 +103,39 @@ runs:
           run: |
               pytest posthog -m "ee" --durations=100 --durations-min=1.0
 
+        - uses: EndBug/add-and-commit@v9 # You can change this to use a specific version.
```

Comment:
Suggestion: Let's move this under .github/workflows/ci-backend.yml and also avoid running this on master.